docker/mariadb/setup.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+echo "Setting up MariaDB databases"
+
+if [ -v OLD_MYSQL_ROOT_PASSWORD ]; then
+    # NB: If the root user already has a password, the init scripts in the
+    # official MariaDB Docker image won't change it, even if
+    # `MYSQL_ROOT_PASSWORD` is different. So we need to do it ourselves.
+    echo "Change old root password"
+    echo "ALTER USER 'root'@'%' IDENTIFIED BY '$MYSQL_ROOT_PASSWORD'; ALTER USER 'root'@'localhost' IDENTIFIED BY '$MYSQL_ROOT_PASSWORD';" \
+        | mysql -u root "-p$OLD_MYSQL_ROOT_PASSWORD"
+fi
+
+i=0
+while true; do
```

Comment:
Minor: the while loop could be a little more idiomatic Bash, and more clear:

```bash
i=0

while [ $i -le 2 ]
do
  echo Number: $i
  ((i++))
done
```
Or maybe even better:

```bash
#!/bin/bash
for i in `seq 3`
do
 ....
done
```

Reply:
I guess in the way you have it, you can add more databases in the environment, without touching the loop, not sure if that could be more elegant in Bash.

Reply:
The issue here is that I am scanning environment variables until I find one that is not defined, so I can't know in advance how many times I need to iterate. I could add an environment variable with the number of databases to create, but I think that would be cumbersome and would add an opportunity to misconfigure things.